samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js

```diff
@@ -167,18 +169,24 @@ const StorybookDecorator = (story, config = {}) => {
 		classes.debug = true;
 	}
 
+	globals.locale = select('locale', locales, Config, globals.locale);
```

Comment:
What's for the 4th param..?

Reply:
It is the selected value. So when we change the story, a global locale is applied.